src/game/client/swarm/c_rd_boss_bar.h

```diff
@@ -24,10 +24,15 @@ class C_RD_Boss_Bar : public C_BaseEntity
 	float m_flBarFlashSustain;
 	float m_flBarFlashInterpolate;
 	bool m_bEnabled;
+	float m_flBarRadius;
+	bool m_bBarTooFarAway = false;
```

Comment:
We usually initialize member variables in the constructor; move the = false there for consistency.